public/apps/EmberApps/Fotoliste/App/HtmlTemplates/Components/personDetailComponent.html

```diff
@@ -3,7 +3,7 @@
 {{else}}
 <div class="photolist__details__close">{{icon "close"}}</div>
 <div class="photolist__details__container">
-    <h2>{{studentData.student.title}}</h2>
```

Comment:
Könntest du hier nicht den Style anpassen statt ein h3 draus zu machen? Es gibt kein weiteres h2 weiter oben in der Hierarchie, soweit ich gesehen habe...